### PR TITLE
fix wording to 'new shared call'

### LINKED ISSUE #134: Using the search field, the »Create a new public room« always is the first result
The entry should not be in there when searching. Otherwise it will always be the first result and instead of starting a call with the best result on pressing enter, a new public room will be created.

cc @nickvergessen

### PR DESCRIPTION
**Pull request by jancborchardt**
We do not use the terms »Room« or »Public« anywhere. So »New shared call« is better than the long »Create a new public room«.

- [x] Fix #134 

Please review @nextcloud/designers @LukasReschke @nickvergessen @MorrisJobke

### CODE REVIEW

**Comment by skjnldsv** [comment]
👍

File: js/app.js [hunk 1]
```diff
@@ -59,7 +59,7 @@
 						var results = [];
 
 						//Add custom entry to create a new empty room
-						results.push({ id: "create-public-room", displayName: t('spreed', 'Create a new public room'), type: "createPublicRoom"});
+						results.push({ id: "create-public-room", displayName: t('spreed', 'New shared call'), type: "createPublicRoom"});
```
Comment (nickvergessen) [inline js/app.js]: I think most people will not understand what this means, click on this and then add people to their call, not knowing, that the call is publicly available if people guess the url....
Sharing is unclear in terms of being public or "i want to add users to this".

Comment (jancborchardt) [inline js/app.js]: Hmm … true. Will change it to »New public room« and we can see how it fares.

**Comment by jancborchardt** [comment]
Adjusted again.

**Comment by jancborchardt** [comment]
@Ivansss fix of the search issue looks good! :+1:

**Pull request merged**